src/EmployerPR/SFA.DAS.EmployerPR.Api/Controllers/ProvidersController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerPR.Application.Queries.GetRoatpProviders;
+
+namespace SFA.DAS.EmployerPR.Api.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class ProvidersController : ControllerBase
```

Comment:
Use class level constructors

Reply:
same issue as the other one, Visual Studio states 'Class cannot have primary constructors' and will not compile. Leaving as is.